### PR TITLE
Fix remove go work 97

### PR DESCRIPTION
**Pull request by logicalangel**

### CODE REVIEW

**Comment by logicalangel** [comment]
depend on #107 
and resolve #119

File: docker/Dockerfile [hunk 1]
```diff
@@ -13,7 +13,7 @@ RUN apt-get update && \
     apt-get upgrade -y && \
     apt-get install --no-install-recommends -y \
     curl ca-certificates && \
-    curl -sSL https://github.com/KenshiTech/unchained/releases/download/${UNCHAINED_VERSION}/unchained.${TARGETOS}.${TARGETARCH} -o unchained && \
+    curl -sSL https://github.com/KenshiTech/unchained/internal/releases/download/${UNCHAINED_VERSION}/unchained.${TARGETOS}.${TARGETARCH} -o unchained && \
```
Comment (pouya-eghbali) [inline docker/Dockerfile:16]: this one should have /internal/

File: Readme.md [hunk 2]
```diff
@@ -2,16 +2,16 @@
 
 # Kenshi Unchained
 
-[![Release](https://shields.io/github/v/release/kenshitech/unchained)](https://github.com/KenshiTech/unchained/releases)
+[![Release](https://shields.io/github/v/release/kenshitech/unchained)](https://github.com/KenshiTech/unchained/internal/releases)
```
Comment (pouya-eghbali) [inline Readme.md:5]: This should not have /internal/

File: Readme.md [hunk 3]
```diff
@@ -2,16 +2,16 @@
...
 [![CLA assistant](https://cla-assistant.io/readme/badge/KenshiTech/unchained)](https://cla-assistant.io/KenshiTech/unchained)
-[![Image tags](https://ghcrbadge-1-y2886489.deta.app/kenshitech/unchained/tags?color=%2344cc11&ignore=latest&n=1&label=image+tags&trim=)](https://github.com/KenshiTech/unchained/pkgs/container/unchained)
-[![Image size](https://ghcrbadge-1-y2886489.deta.app/kenshitech/unchained/size?color=%2344cc11&tag=latest&label=image+size&trim=)](https://github.com/KenshiTech/unchained/pkgs/container/unchained)
+[![Image tags](https://ghcrbadge-1-y2886489.deta.app/kenshitech/unchained/tags?color=%2344cc11&ignore=latest&n=1&label=image+tags&trim=)](https://github.com/KenshiTech/unchained/internal/pkgs/container/unchained)
```
Comment (pouya-eghbali) [inline Readme.md:7]: This should not have /internal/

File: Readme.md [hunk 4]
```diff
@@ -2,16 +2,16 @@
...
+[![Image size](https://ghcrbadge-1-y2886489.deta.app/kenshitech/unchained/size?color=%2344cc11&tag=latest&label=image+size&trim=)](https://github.com/KenshiTech/unchained/internal/pkgs/container/unchained)
```
Comment (pouya-eghbali) [inline Readme.md:8]: This should not have /internal/

File: Readme.md [hunk 5]
```diff
@@ -2,16 +2,16 @@
...
 [![Open Collective backers and sponsors](https://img.shields.io/opencollective/all/unchained)](https://opencollective.com/unchained)
 [![Documentation](https://img.shields.io/badge/Documentation-brightgreen)](https://kenshi.io/docs/unchained)
 
 Unchained is a decentralized, federated network for data indexing, validation, analysis, processing, and delivery.
 
-[unchained.demo.webm](https://github.com/KenshiTech/unchained/assets/4604055/427740a2-799c-42e2-a863-f44cc6e74d46)
+[unchained.demo.webm](https://github.com/KenshiTech/unchained/internal/assets/4604055/427740a2-799c-42e2-a863-f44cc6e74d46)
```
Comment (pouya-eghbali) [inline Readme.md:14]: This should not have /internal/

File: Readme.md [hunk 6]
```diff
@@ -37,7 +37,7 @@ See [License](./LICENSE) for more details.
 
 Thank you to all the people who have already contributed to Unchained!
 
-<a href="https://github.com/KenshiTech/unchained/graphs/contributors">
+<a href="https://github.com/KenshiTech/unchained/internal/graphs/contributors">
```
Comment (pouya-eghbali) [inline Readme.md:40]: This should not have /internal/

**Review by pouya-eghbali — CHANGES_REQUESTED** [review]

File: quickstart.md [hunk 7]
```diff
@@ -25,7 +25,7 @@ You can setup a node using a docker deployment or local deployment:
 Running Unchained with Docker is straightforward. Make sure you
 [have Docker installed](https://docs.docker.com/engine/install/).
 Then, head over to the
-[Unchained release page](https://github.com/KenshiTech/unchained/releases)
+[Unchained release page](https://github.com/KenshiTech/unchained/internal/releases)
```
Comment (pouya-eghbali) [inline quickstart.md:28]: This one shouldn't have /internal/

**Review by pouya-eghbali — CHANGES_REQUESTED** [review]

**Pull request merged**